lib/un_locode/version.rb

```diff
@@ -1,3 +1,3 @@
 module UnLocode
-  VERSION = '0.0.4'
+  VERSION = '0.0.5'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.